rotkehlchen/chain/evm/accounting/interfaces.py

```diff
@@ -36,3 +46,68 @@ def event_settings(self, pot: 'AccountingPot') -> dict[str, 'TxEventSettings']:
     def reset(self) -> None:  # pylint: disable=no-self-use
         """Subclasses may implement this to reset state between accounting runs"""
         return None
+
+
+class DepositableAccountantInterface(ModuleAccountantInterface):
+    """
+    Interface for protocols that allow to deposit multiple tokens in exchange for a token
+    representing the position in the pool. Examples are:
+    - Curve
+    - Balancer
+    """
+
+    def _process_deposit_or_withdrawal(
+            self,
+            pot: 'AccountingPot',
+            event: 'HistoryBaseEntry',
+            other_events: Iterator['HistoryBaseEntry'],
+    ) -> None:
+        """
+        Process deposits and withdrawals from protocols that allow to deposit multiple assets
+        in return for a wrapped token. There are multiple events that we have to consume from
+        the iterator.
+
+        The receive wrapped event needs to have in the extra data field a key `deposit_events_num`
+        marking the number of events to consume.
+        The return wrapped event needs to have the key `withdrawal_events_num` with the number
+        of events in the return event.
+        """
+        method: Literal['acquisition', 'spend']
+        if event.event_type == HistoryEventType.RECEIVE:
+            # Pool token is received, which means it is a deposit
+            events_to_consume = event.extra_data.get('deposit_events_num', None) if event.extra_data is not None else None  # noqa: E501
+            method = 'spend'
+        else:  # Withdrawal
+            events_to_consume = event.extra_data.get('withdrawal_events_num', None) if event.extra_data is not None else None  # noqa: E501
+            method = 'acquisition'
+
+        if events_to_consume is None:
+            log.debug(
+                f'Could not find the number of events to consume for a curve deposit/withdrawal '
```

Comment:
You need the name here, so this needs a constructor .. or you can use the counterparty as you did below

```suggestion
                f'Could not find the number of events to consume for a {self.name} deposit/withdrawal '
```